### ISSUE TITLE
[Feature Request] Dismiss Button within Notification

### ISSUE THREAD

**Issue by GuteIdee** [khuttun/NotificationNotes#24]
A Dismiss Button at the Notification in the NotificationBar to disable the Note (without opening the app) would be nice.

**Comment by khuttun** [comment]
Good idea. A similar design as with the calendar reminder notification Done button might work.

**Comment by diegoalvis** [comment]
I will take this issue! 😄
